index.html

```diff
@@ -0,0 +1,207 @@
+              <img class="icon" src="assets/linked.png" alt="Linkedin image" />
+            </a>
+          </li>
+        </ul>
+      </section>
+      <section class="recent-works">
+        <div class="grid-container">
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-me-section">
+        <div class="about-me">
+          <h2>About me</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
```

Comment:
- [ ] Kindly make sure your about me section has the right background image and uses the same colours as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177016153-332f489b-1fd7-4584-ac00-8c137e5981e4.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177016161-8234d8d1-9e5e-4109-ac5b-b74c871312cc.png)